Marlin/src/feature/probe_temp_comp.h

```diff
@@ -33,10 +33,10 @@ enum TempSensorID : uint8_t {
 };
 
 typedef struct {
-  uint8_t measurements; // Max. number of measurements to be stored (35 - 80°C)
-  float   temp_res,     // Resolution in °C between measurements
-          start_temp,   // Base measurement; z-offset == 0
-          end_temp;
+  uint8_t measurements;       // Max. number of measurements to be stored (35 - 80°C)
+  celsius_float_t temp_res,   // Resolution in °C between measurements
+                  start_temp, // Base measurement; z-offset == 0
+                  end_temp;
```

Comment:
if this is `float`, this should be float too: `ProbeTempComp::compensate_measurement(const TempSensorID tsi, const celsius_t temp, float &meas_z) `, I guess...